New log file opened at Tue Apr 11 23:10:40 CEST 2017 by 1132@Martin-pc
Taking next connection to: 56023
New client attempting to connect on port: 56024-56023
  Address = /192.168.56.1:56023
This client on port 56024 has uuid 4f06af40-d440-4230-b64a-5624a574d1c9 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-4f06af40-d440-4230-b64a-5624a574d1c9
Taking next connection to: 56023
New client attempting to connect on port: 56026-56023
  Address = /192.168.56.1:56023
Handshake exception on socket: 56026
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:150)
	at java.net.SocketInputStream.read(SocketInputStream.java:121)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 56023
New client attempting to connect on port: 56031-56023
  Address = /192.168.56.1:56023
This client on port 56031 has uuid c33872fc-7b80-46f0-9f71-0519a6764349 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-4f06af40-d440-4230-b64a-5624a574d1c9, activator-c33872fc-7b80-46f0-9f71-0519a6764349
Taking next connection to: 56023
New client attempting to connect on port: 56063-56023
  Address = /192.168.56.1:56023
Handshake exception on socket: 56063
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:150)
	at java.net.SocketInputStream.read(SocketInputStream.java:121)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 56023
New client attempting to connect on port: 56064-56023
  Address = /192.168.56.1:56023
This client on port 56064 has uuid 7d8d5449-6256-497d-b3ab-77d0163d94de configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-4f06af40-d440-4230-b64a-5624a574d1c9, play-fork-run-7d8d5449-6256-497d-b3ab-77d0163d94de
Taking next connection to: 56023
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 56023
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 56023
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 56023
